dev-lang/erlang/files/epmd.init-r1

```diff
@@ -0,0 +1,26 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the Erlang Public License 1.1
+
+ZT_ADDRESS=""
+pidfile="/var/run/epmd.pid"
+command_args="${command_args}"
+
+depend() {
+	need loopback
+	before sshd
+}
+
+start() {
+	ebegin "Starting Erlang Port Mapper Daemon"
+	start-stop-daemon --start --quiet \
+		--pidfile $pidfile \
```

Comment:
While at it, please quote `${pidfile}`, and always use `${foo}` braced form for variables.

Reply:
fix

Reply:
It's still not quoted.

Reply:
https://github.com/gentoo/gentoo/blob/8aa2bb6a520ee169c17be513254f5099e7644a98/dev-lang/erlang/files/epmd.init-r1#L15

Reply:
There's no quoting (`""`) there.

Reply:
> quoting

sorry. fixed